src/html_toast.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * Configuration options for toasts.
+ */
+interface HtmlToastOptions extends Blockly.ToastOptions {
```

Comment:
Did you intend to use this? It appears unused, so I'm wondering if it should be removed or if `HtmlToast` should be updated to use it.